src/main/api/createAPIKey.json

```diff
@@ -0,0 +1,28 @@
+{
+  "uri": "/api/apikey/key/",
+  "comments": [
+    "Creates an authenticatio API key. You can optionally specify an Id for the application, if not provided one will be generated."
```

Comment:
We should probably describe some of the contract here with regards to how we create an API key. 

For example, an API key can only be created with equal or lesser authority. An API key cannot create another API key unless it is granted to that API key. 

If an API key is locked to a tenant, it can only create API Keys for that same tenant.  Etc.